include/seqan3/alignment/pairwise/execution/alignment_executor_two_way.hpp

```diff
@@ -56,19 +57,19 @@ class alignment_executor_two_way
     /*!\name Resource types
      * \{
      */
-    //!\brief The underlying resource type.
-    using resource_type = decltype(views::zip(std::declval<resource_t>(), std::views::iota(0)) | views::chunk(1));
+    //!\brief The underlying resource type augmented with an index and a chunked view.
+    using chunked_resource_type = typename chunked_indexed_sequence_pairs<resource_t>::type;
     //!\brief The iterator over the underlying resource.
-    using resource_iterator_t = std::ranges::iterator_t<resource_type>;
-    //!\brief The value type of the resource.
-    using resource_value_type = std::ranges::range_value_t<resource_type>;
+    using chunked_chunked_resource_iterator_t = std::ranges::iterator_t<chunked_resource_type>;
```

Comment:
sometimes you abbreviate with `_t` and sometimes you call it `_type` (`chunked_resource_type` vs `chunked_chunked_resource_iterator_t`)
Maybe change one of them to match the others? (wwithout chenging the whole file but only the lines you touch anyway)

Reply:
right. I kept the naming close to stl. So iterator without any suffix and everything else with _type suffix.